src/build-cli.ts

```diff
@@ -7,6 +7,8 @@ import { commonArgs } from "./util"
 import { printErrorAndExit } from "./promise"
 import { tsAwaiter } from "./awaiter"
 import cla = require("command-line-args")
```

Comment:
Not possible. It is not es6 default export. TS requires explicit `default` in the raw code.